addons/mail/static/src/core/common/persona_service.js

```diff
@@ -89,6 +89,14 @@ export class PersonaService {
         });
     }
 
+    async openUserPreferences() {
+        if (this.store.user?.user?.id) {
+            const actionDescription = await this.orm.call("res.users", "action_get");
+            actionDescription.res_id = this.store.user.user.id;
+            this.env.services.action.doAction(actionDescription);
+        }
+    }
+
```

Comment:
Nitpicking: better keep this in component, as it's used only once, and it feels slightly off that current user preferences features are on persona service.

Eventually we'd like to remove discuss services whose name and methods are too coupled with a specific model. Very likely `PersonaService` will be removed in the near future.